### ISSUE TITLE
Can't restart after server crash

### ISSUE THREAD

**Issue by dougharris** [NodeMC/NodeMC-CORE#2]
I'm running spigot 1.9 and had a crash. I tried both the stop and restart buttons on the NodeMC UI but neither worked. It looks like this is because both of those assume that the server is still running and NodeMC tries to write to the existing log output:

```
 serverSpawnProcess.stdin.write('say [NodeMC] Restarting server!\n');
```

**Comment by gmemstr** [comment]
Will investigate, thanks for raising the issue. This didn't come up in my testing. Expect a bugfix by tomorrow.

**Comment by gmemstr** [comment]
Fixed as of commit [#25f8182](https://github.com/NodeMC/NodeMC-CORE/commit/25f8182f52f2ed7ce5182696f25c9b5aaf228914)
